snap/info_snap_yaml_test.go

```diff
@@ -788,3 +790,124 @@ slots:
 	c.Check(slot2.Apps, DeepEquals, map[string]*snap.AppInfo{
 		app1.Name: app1, app2.Name: app2})
 }
+
+// type and architectures
+
+func (s *YamlSuite) TestSnapYamlTypeDefault(c *C) {
+	y := []byte(`name: fatbinary
```

Comment:
I feel a bit bad about writing this as this is super nitpick land - this is not a fatbinary, the default here is arch: all which may be a shell script or something else. But given that its a test and is not really used its not really important at all.